### PR TITLE
Wifibroadcast enhancements

### PR DESCRIPTION
**Pull request by hamishwillee**
This updates #321
- Fixes some typos
- Consistent heading capitalisation
- Consistent naming of items
- Restructure.

The main restructure is to introduce the aims of wifibroadcast and purpose early. Then all the rest of the documentation  and explanation is pushed into the FAQ.

### CODE REVIEW

File: en/qgc/video_streaming_wifi_broadcast.md [hunk 1]
```diff
@@ -1,208 +1,189 @@
...
-     * 1GB SDRAM
+#### What ARM Boards can be used for UAV
 
-   Cons:
-     * Very sensitive to radio interference
-     * Doesn't supported by mainline kernel
-     * High cost
-     * HW video encoder is broken
-     * Bad PCB quality (too thin, ground pins without solder bridges)
+Board | Pros  | Cons
```
Comment (hamishwillee) [inline en/qgc/video_streaming_wifi_broadcast.md:170]: FYI, this renders for easier comparison of the different boards.

File: en/qgc/video_streaming_wifi_broadcast.md [hunk 2]
```diff
@@ -1,208 +1,189 @@
...
 ## FAQ
-Q: What is a difference from original wifibroadcast?
 
-A: Original version of wifibroadcast use a byte-stream as input and splits it to packets of fixed size (1024 by default). If radio
-packet was lost and this is not corrected by FEC you'll got a hole at random (unexpected) place of stream. This is especially bad if
-data protocol is not resistant to (was not desired for) such random erasures. So i've rewrite it to use UDP as data source and pack one
-source UDP packet into one radio packet. Radio packets now have variable size depends on payload size. This is reduces a video latency a lot.
 
-Q: What type of data can be transmitted using wifibroadcast?
+#### What are the limitations of normal WiFi for long-distance video transfer?
```
Comment (hamishwillee) [inline en/qgc/video_streaming_wifi_broadcast.md:113]: Pushed this down into FAQ.

**Comment by hamishwillee** [comment]
@svpcom This updates your doc to improve the structure. Essentially the top is for essential introduction and how to, while the bottom is interesting discussion (ie FAQ). The TODO are also at the bottom so as not to interfere with the more important text. 

Can you please sanity check to confirm you are OK with the changes, and that I didn't break anything!

**Comment by svpcom** [comment]
@hamishwillee I've found two minor issues:
1. "solder bridges" need to be replaced by [thermal relief](https://en.wikipedia.org/wiki/Thermal_relief)
2. Need to add a dot after "See wiki article" in enhanced setup chapter.

**Comment by hamishwillee** [comment]
Thanks very much @svpcom . I have fixed those. Anything else, or are you happy for me to merge?

**Comment by svpcom** [comment]
I think article is ready

**Comment by hamishwillee** [comment]
Thanks!

**Comment by svpcom** [comment]
@hamishwillee 
Another minor issue: "This setup uses WiFi in unconnected (broadcast) mode and software from the Wifibroadcast project." where link is set to original wifibroadcast which is not supported by author anymore. This may confuse users because my version is not derive any code from it and only shares project name. I think that link should be https://github.com/svpcom/wifibroadcast/wiki

**Comment by hamishwillee** [comment]
Thanks. Fixed in https://github.com/PX4/Devguide/commit/a3f94ee5135f03e8a1f5e24e147700615efced8f

**Comment by hamishwillee** [comment]
I also clarified the "original project" a bit more in the FAQ - it was a bit odd because we referred to the original project in a FAQ and link, but didn't explain anything about that project. The update makes it clear they share the same name, but no code: https://github.com/PX4/Devguide/commit/21b94adfd4dd5d8a15612575e8eff169bfd84f42

**Pull request merged**